src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolution.cs

```diff
@@ -83,6 +85,12 @@ internal async Task RemoveAllRenameAnnotationsAsync(IEnumerable<DocumentId> docu
             _newSolution = _intermediateSolutionContainingOnlyModifiedDocuments;
         }
 
+        internal void RenameDocumentToMatchNewSymbol(Document document)
+        {
+            var documentName = $"{ReplacementText}{document.Project.GetLanguageSourceFileExtension()}";
```

Comment:
You'll want to reuse the existing extension in the case of .cs/.csx scenarios (if we care about those.)